smartapps/lunkwillandfook/myhouse-for-amazon-echo.src/myhouse-for-amazon-echo.groovy

```diff
@@ -0,0 +1,612 @@
+/**
+ *  myHouse for Amazon Echo
+ *
+ *  Copyright 2015 Jeremy Huckeba
+ *  Version 1.00 1/18/16
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  Ties a Hello, Home phrase to a switch's (virtual or real) on/off state. Perfect for use with IFTTT.
+ *  Simple define a switch to be used, then tie the on/off state of the switch to a specific Hello, Home phrases.
+ *  Connect the switch to an IFTTT action, and the Hello, Home phrase will fire with the switch state change.
+ *
+ *
+ */
+definition(
+    name: "myHouse for Amazon Echo",
+    namespace: "LunkwillAndFook",
+    author: "Jeremy Huckeba",
+    description: "Allows deep, custom integration with Amazon Echo.",
+    category: "Convenience",
+    iconUrl: "https://s3.amazonaws.com/myhouseapp/st/my-house-src-st.png",
+    iconX2Url: "https://s3.amazonaws.com/myhouseapp/st/my-house-src-st@2x.png",
+    iconX3Url: "https://s3.amazonaws.com/myhouseapp/st/my-house-src-st@3x.png")
+
+def LEAVINGPHRASETITLE = "I'm leaving..."
+def WATCHINGMOVIEPHRASETITLE = "I'm watching a movie..."
+def FINISHEDWATCHINGMOVIEPHRASETITLE = "I'm finished watching a movie..."
+def HAVINGPARTYPHRASETITLE = "I'm having a party..."
+def BACKPHRASETITLE = "I'm back..."
+def CLEANINGPHRASETITLE = "I'm cleaning..."
+def SLEEPPHRASETITLE = "I'm going to sleep..."
+def AWAKEPHRASETITLE = "I'm awake..."
```

Comment:
Defining constants like this won't work, see the [docs](http://docs.smartthings.com/en/latest/getting-started/groovy-for-smartthings.html#global-variables) for more info.